### PR TITLE
Added ${aspnet-response-statuscode} - rendering response status code

### PR DESCRIPTION
**Pull request by ThomasArdal**

### CODE REVIEW

File: src/Shared/LayoutRenderers/AspNetResponseStatusCodeRenderer.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            if (httpResponse == null)
+            {
+                return;
+            }
+
+            int statusCode;
+#if !ASP_NET_CORE
+            statusCode = httpResponse.StatusCode;
+#else
+            statusCode = httpResponse.StatusCode;
```
[the remark this answers is not in the document]
Comment (304NotModified) [reply src/Shared/LayoutRenderers/AspNetResponseStatusCodeRenderer.cs]: I fixed this :)

**Comment by 304NotModified** [comment]
@ThomasArdal 4.9 is released! (indexing now on Nuget). Thanks for the contributions!

**Comment by 304NotModified** [comment]
done by https://github.com/NLog/NLog.github.io/pull/129

**Comment by ThomasArdal** [comment]
garfield

**Pull request merged**